creator-node/src/components/healthCheck/healthCheckAPI.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const { handleResponse, successResponse } = require('../../apiHelpers')
+const { healthCheck } = require('./healthCheck')
+const { serviceRegistry } = require('../../serviceRegistry')
+
+const router = express.Router()
+
+// Controllers
+
+const healthCheckController = async () => {
```

Comment:
not super clear on separation between this controller and the actual healthcheck component. works fine here bc it's so simple but how would this look in more complex routes that return response in multiple codepaths? do you have some switch statement in the controller to handle each potential response?

side note, terminology? how do i refer to the "healthcheck" component to distinguish it from the "healthcheckcontroller" component

Reply:
Good q - the idea is for controllers to map 1:1 with routes. A controller should be responsible for routing level concerns (param validation/munging, error handling, response codes etc), delegating the actual logic to business logic functions. So other routes like `health_check/ipfs` would have their own controller. Probably would have been helpful to pull out another route to make this clearer.

terminology question is a great one, we've been wrestling with this a bit on the Discprov refactor as well. Maybe it should be `healthCheckService.js`? Or we just refer to it as healthcheck business logic? wdyt

Reply:
kk so `healthCheckComponentService` right?

Reply:
yep!

Reply:
👍